torch/utils/hipify/hipify_python.py

```diff
@@ -802,9 +808,10 @@ def hipify(
     all_files = list(matched_files_iter(output_directory, includes=includes,
                                         ignores=ignores, extensions=extensions,
                                         out_of_place_only=out_of_place_only))
+    all_files += [f for f in extra_files if f not in all_files]
```

Comment:
quadratic police!

Reply:
He. Yeah. I figured the enumeration would be short (because usually the extra_files is empty), but I'll go for a set.